src/lib/datepicker/datepicker-input.ts

```diff
@@ -124,6 +125,17 @@ export class MdDatepickerInput<D> implements AfterContentInit, ControlValueAcces
   }
   private _max: D;
 
+  /** Whether the datepicker-input is disabled. */
+  @Input()
```

Comment:
you'll need to also add a host binding: `'[disabled]': 'disabled'`, to keep the native disabled attribute in sync with this